WebCryptoAPI/import_export/importKey_failures.js

```diff
@@ -136,20 +137,20 @@ function run_test(algorithmNames) {
     // Should fail due to SyntaxError
     testVectors.forEach(function(vector) {
         var name = vector.name;
-        validKeyData.filter((test) => test.format === 'pkcs8' || (test.format === 'jwk' && test.data.d)).forEach(function(test) {
-            allAlgorithmSpecifiersFor(name).forEach(function(algorithm) {
+        allAlgorithmSpecifiersFor(name).forEach(function(algorithm) {
+            validKeyData[dataKey(algorithm)].filter((test) => test.format === 'pkcs8' || (test.format === 'jwk' && test.data.d)).forEach(function(test) {
```

Comment:
Instead of test.data.d -> isPrivateKey? I.e. a function taking a data and checking inside if d is present?

Reply:
I'm not opposed, bit given that it's only used there, do you think it's worth ?

Reply:
My usual logic is that I don't want to remember what exactly "d" means and why I care about it the next time when I read the code.

If you think that it makes reading the code more annoying, I am totally ok with letting it be as now

Reply:
fair enough, I'll add that function.